screens/SearchProductScreen.jsx

```diff
@@ -0,0 +1,102 @@
+import { StyleSheet, View, Text, Image, ImageBackground, ScrollView } from "react-native";
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+const SearchProductScreen = () => {
+  let [Products, setProducts] = useState([]);
```

Comment:
Use `const` instead of `let` cause all states should be hosted in constant. In addition, variables in Javascript are written using camelCase